src/glfw-sandbox/drawings.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+#include <windows.h>
+#include <GL/gl.h>
+#include <GL/glu.h>
+#include <math.h>
+
+
+class drawObjects
```

Comment:
Tip: What is the casing convention for C++ classes? Does this adhere to it? :)

Reply:
CamelCase? like "DrawObjects"?

Reply:
The current code has camelCase, which means first letter small.

I think for classes it is more common to use PascalCase, which is that DrawObjects you suggest :)

...and yes,